client/src/components/Community/Community.js

```diff
@@ -9,9 +9,9 @@ function Community() {
   return (
     <div className="community-component">
       <div className="side-banner">Wings of House of Geeks</div>
+      <div className="title">Communities of House of Geeks</div>
       <div className="container-fluid main">
-        <div className="row justify-content-around">
-          <div className="title">Communities of House of Geeks</div>
```

Comment:
Retain this. Don't add a new tag for this. Try to manipulate the present tags and classes to solve this issue

Reply:
I have erased the tap and space.Just add a judge condition. Thanks.